Automattic/studio
# Studio: Remove wp-content on import

Fixes https://github.com/Automattic/dotcom-forge/issues/8713

## Proposed Changes

This PR removes the `wp-content` folder on import to  avoid merging the current content with the incoming imported content.
NOTE: We keep `database`, `db.php` and `mu-plugins` folder to avoid messing up with `sqlite-integration-plugin` and the rest of `mu-plugins` that are needed by Studio

## Testing Instructions
1. Create a new site.
2. Ensure that you have 2 backups. 1 should have less plugins installed than the other. ( Same can be done with themes , this is an easy way to check if we merging things or not ).
3. Import each on of the backups sequentially. Ensure that the last's site imported plugins are preserved.
4. Start/Stop the server, and ensure everything works as expected


## Pre-merge Checklist
